crates/client/assets/schema.sdl

```diff
@@ -1242,6 +1250,12 @@ scalar U64
 
 union UpgradePurpose = ConsensusParametersPurpose | StateTransitionPurpose
 
+type UploadedBytecode {
+	bytecode: HexString!
+	uploadedSubsectionsNumber: Int!
```

Comment:
not too clear if this means - number of subsections uploaded, or the index of the uploaded subsections

Reply:
Good point. I'll add a docstring so it is clear that it is the former interpretation we refer to.

Reply:
Done. I also made this field optional since it should not be set if the upload is completed.

Reply:
nice